src/caselawclient/models/utilities/dates.py

```diff
@@ -9,11 +9,7 @@ def parse_string_date_as_utc(iso_string: str, timezone: tzinfo.BaseTzInfo) -> da
     ensure that it is converted to a UTC-aware datetime"""
 
     mixed_date = isoparse(iso_string)
-    if not mixed_date.tzinfo:
-        # it is an unaware time
-        aware_date = timezone.localize(mixed_date)
-    else:
-        aware_date = mixed_date
+    aware_date = timezone.localize(mixed_date) if not mixed_date.tzinfo else mixed_date
```

Comment:
I'd be tempted to invert this -- `... = mixed_date if mixed_date.tzinfo else timezone.localize(...)`